serverless.yml

```diff
@@ -6,7 +6,7 @@ provider:
   stage: ${opt:stage, 'TEST'}
   runtime: java11
   memorySize: 768
-  timeout: 60
+  timeout: 180
```

Comment:
This should avoid timeouts related to large location folders like California and Nebraska.